### PR TITLE
Kerlink Wirnet connecting updates, add iStation

### PR DESCRIPTION
**Pull request by rvolosatovs**
#### Summary

References #1140 https://github.com/TheThingsNetwork/kerlink-wirnet-firmware/pull/11

#### Changes

- Bring Kerlink Wirnet Station connection guide up-to-date with changes made by https://github.com/TheThingsNetwork/kerlink-wirnet-firmware/pull/11
- Add Kerlink Wirnet iStation connection guide

#### Notes for Reviewers

iStation guide is just a copy-paste of Station guide with some stuff removed and gateway model replaced accordingly
Setting priority to `medium`, since https://github.com/TheThingsNetwork/lorawan-stack/issues/1140 is `high`, but the iStation gateway itself is not present in the OP.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`.
- [x] Testing: The changes are covered with unit tests. The changes are tested manually as well.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by rvolosatovs** [comment]
Blocked by https://github.com/TheThingsNetwork/kerlink-wirnet-firmware/pull/11.
Will mark "ready for review" once https://github.com/TheThingsNetwork/kerlink-wirnet-firmware/pull/11 is merged.

File: doc/content/guides/connecting-gateways/kerlinkwirnetistation/_index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Kerlink Wirnet iStation"
+description: ""
+weight: 1
+---
+
+Kerlink Wirnet Station is a LoRaWAN gateway, whose technical specifications can be found in [the official documentation](https://www.kerlink.com/product/wirnet-istation/). This page guides you to connect it to {{% tts %}}.
```
Comment (johanstokking) [inline doc/content/guides/connecting-gateways/kerlinkwirnetistation/_index.md]: Isn't this the iStation then?

Isn't this extremely confusing to have Station and iStation?

Comment (rvolosatovs) [reply doc/content/guides/connecting-gateways/kerlinkwirnetistation/_index.md]: > Isn't this the iStation then?

It is indeed. Will fix.

> Isn't this extremely confusing to have Station and iStation?

It is indeed. I doubt Kerlink will rename the gateway though, so we have to deal with it.

**Comment by rvolosatovs** [comment]
Setting priority to *high*, since this needs to land in a release this week for a customer

**Pull request merged**